packages/framework/src/Model/Inquiry/Inquiry.php

```diff
@@ -107,7 +107,7 @@ class Inquiry
      */
     public function __construct(InquiryData $inquiryData)
     {
-        $this->createdAt = $inquiryData->createdAt ?? new DateTimeImmutable();
+        $this->createdAt = $inquiryData->createdAt ?? Clock::get()->now();
         $this->domainId = $inquiryData->domainId;
```

Comment:
Why left the default value in this entity, when we removed it in the articles, blog articles, etc.